[2024-08-10T11:39:50.241+0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-10T11:39:50.259+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pollution_air_dags.extract_air_pollution manual__2024-08-10T08:39:44.020473+00:00 [queued]>
[2024-08-10T11:39:50.269+0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pollution_air_dags.extract_air_pollution manual__2024-08-10T08:39:44.020473+00:00 [queued]>
[2024-08-10T11:39:50.270+0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-10T11:39:50.297+0300] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_air_pollution> on 2024-08-10 08:39:44.020473+00:00
[2024-08-10T11:39:50.304+0300] {standard_task_runner.py:64} INFO - Started process 7752 to run task
[2024-08-10T11:39:50.310+0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'pollution_air_dags', 'extract_air_pollution', 'manual__2024-08-10T08:39:44.020473+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/air_pollution_dags.py', '--cfg-path', '/tmp/tmpy1z0zsme']
[2024-08-10T11:39:50.313+0300] {standard_task_runner.py:91} INFO - Job 102: Subtask extract_air_pollution
[2024-08-10T11:39:50.395+0300] {task_command.py:426} INFO - Running <TaskInstance: pollution_air_dags.extract_air_pollution manual__2024-08-10T08:39:44.020473+00:00 [running]> on host DESKTOP-DKUGVOH.
[2024-08-10T11:39:50.565+0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='pollution_air_dags' AIRFLOW_CTX_TASK_ID='extract_air_pollution' AIRFLOW_CTX_EXECUTION_DATE='2024-08-10T08:39:44.020473+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-10T08:39:44.020473+00:00'
[2024-08-10T11:39:50.570+0300] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-10T11:39:50.572+0300] {logging_mixin.py:188} INFO - Le fichier 'air_pollution_data.csv' est introuvable. Assurez-vous qu'il est bien dans le mÃªme dossier que ce script.
[2024-08-10T11:39:50.572+0300] {python.py:237} INFO - Done. Returned value was: []
[2024-08-10T11:39:50.573+0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-10T11:39:50.602+0300] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pollution_air_dags, task_id=extract_air_pollution, run_id=manual__2024-08-10T08:39:44.020473+00:00, execution_date=20240810T083944, start_date=20240810T083950, end_date=20240810T083950
[2024-08-10T11:39:50.645+0300] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-10T11:39:50.672+0300] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-10T11:39:50.687+0300] {local_task_job_runner.py:222} INFO - ::endgroup::
